internal/connect/command_offset_describe.go

```diff
@@ -35,23 +35,22 @@ type metadataDescribeOut struct {
 
 func (c *offsetCommand) newDescribeCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:               "get <id>",
-		Short:             "Get connector offsets",
+		Use:               "describe <id>",
+		Short:             "describe connector offsets",
 		Args:              cobra.ExactArgs(1),
 		RunE:              c.getOffset,
 		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
 		Annotations:       map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
 		Example: examples.BuildExampleString(
 			examples.Example{
-				Text: "Get connector offsets for a lccId in the current or specified Kafka cluster context.",
-				Code: "confluent connect offset get lcc-123456",
+				Text: "Describe offsets for a connector in the current or specified Kafka cluster context.",
```

Comment:
Is the proper terminology "connect cluster"? If I remember correctly, lcc stands for "logical connect cluster".

Reply:
Other API seems to write connector and thats is the preffered term.  `Describe offsets for a connector` is okay